desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -96,22 +96,33 @@ def handle_on_message(channel_id, bot_id, elements, user_id):
       _send_message(channel_id, bot_message)
 
     if text.lower().startswith('select'):
-      detect_select_statement(channel_id, user_id, text.lower())
-
-
-def detect_select_statement(channel_id, user_id, statement):
-  if not statement.endswith(';'):
-    statement += ';'
-  
-  raw_sql_message = 'Hi <@{user}>\nLooks like you are copy/pasting SQL, instead now you can send Editor links which unfurls in a rich preview!'.format(user=user_id)
-  _send_ephemeral_message(channel_id, user_id, raw_sql_message)
-
+      detect_select_statement(host_domain, is_http_secure, channel_id, user_id, text.lower())
+
+
+def detect_select_statement(host_domain, is_http_secure, channel_id, user_id, statement):
+  slack_user = check_slack_user_permission(host_domain, user_id)
+  user = get_user(channel_id, slack_user)
+
+  gist_doc = Document2.objects.create(
+    name='Mysql Query blah',
+    type='gist',
+    owner=user,
+    data=json.dumps({'statement_raw': statement}),
+    extra='mysql',
```

Comment:
As we don't know the dialect we should default to the default one (if not we lose a lot in usability).

They are ways to detect which type of SQL it is but bit more complex and to see later.